homeassistant/components/input_select.py

```diff
@@ -139,3 +173,19 @@ def select_option(self, option):
             return
         self._current_option = option
         self.update_ha_state()
+
+    def select_next(self):
```

Comment:
The service is a bit more complicated, so agree that they are ok as is.

The reason for this internal method is reducing duplication (The only difference is `+1` / `-1` and that could be easily handled by a parameter)